File: Problemas/1030A - In Search of an Easy Problem/1030A_In_Search_Of_An_Easy_Problem.py
# 1030A - In Search of an Easy Problem
# https://codeforces.com/problemset/problem/1030/A

# Input:
# 1) Número de respuestas
# 2) Respuestas ede cada persona
num_respuestas = int(input())
respuestas = list(map(int, input().split()))

# Si hay un 1 en las respuestas, entonces imprime HARD, de lo contrario, imprime EASY
if 1 in respuestas:
    print('HARD')
else:
    print('EASY')

File: Problemas/1030A - In Search of an Easy Problem/1030A_In_Search_Of_An_Easy_Problem.cpp
// 1030A - In Search of an Easy Problem
// https://codeforces.com/problemset/problem/1030/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) num_respuestas = Número de respuestas
    int num_respuestas;
    cin >> num_respuestas;

    // Checa la respuesta de cada persona
    while(num_respuestas--) {

        // Input:
        // 2) respuesta = Respuesta de la persona
        int respuesta;
        cin >> respuesta;

        // Si la respuesta es igual a 1, entonces imprime 'HARD' y termina el programa
        if(respuesta == 1) {
            cout << "HARD" << endl;
            return 0;
        }
    }

    // Si no hubo respuestas iguales a 1, entonces imprime 'EASY'
    cout << "EASY" << endl;

    return 0;
}
